src/geowombat/core/io.py

```diff
@@ -683,34 +681,38 @@ def to_netcdf(
 def save(
     data: xr.DataArray,
     filename: T.Union[str, Path],
-    mode: T.Optional[str] = "w",
-    nodata: T.Optional[T.Union[float, int]] = None,
-    overwrite: T.Optional[bool] = False,
+    overwrite: bool = False,
+    scatter: T.Optional[str] = None,
     client: T.Optional[Client] = None,
-    compute: T.Optional[bool] = True,
+    compute: bool = True,
     tags: T.Optional[dict] = None,
     compress: T.Optional[str] = "none",
     compression: T.Optional[str] = None,
-    num_workers: T.Optional[int] = 1,
-    log_progress: T.Optional[bool] = True,
+    num_workers: int = 1,
+    log_progress: bool = True,
     tqdm_kwargs: T.Optional[dict] = None,
     bigtiff: T.Optional[str] = None,
 ):
     """Saves a DataArray to raster using rasterio/dask.
 
     Args:
+        data (xarray.DataArray): The data to write.
         filename (str | Path): The output file name to write to.
         overwrite (Optional[bool]): Whether to overwrite an existing file. Default is False.
-        mode (Optional[str]): The file storage mode. Choices are ['w', 'r+'].
-        nodata (Optional[float | int]): The 'no data' value. If ``None`` (default), the 'no data'
-            value is taken from the ``DataArray`` metadata.
+        scatter (Optional[str]): Scatter 'band' or 'time' to separate file. Default is None.
```

Comment:
@jgrss I am not sure what you mean here. Does it create a new file for each band?